source/_posts/2019-10-09-release-100.markdown

```diff
@@ -274,6 +308,50 @@ TBD
 - Add device trigger support to sensor entities ([@emontnemery] - [#27133]) ([automation docs]) ([binary_sensor docs]) ([device_automation docs]) ([sensor docs])
 - Add Google Report State ([@balloob] - [#27112]) ([alexa docs]) ([cloud docs]) ([google_assistant docs])
 
+## All frontend changes
+
+* Fix dark switch for map card editor (#3856) @bramkragten
```

Comment:
I do like having more information about what is happening in the frontend, however, maybe we should update the hass-release repo to support that (and actually generate links for these things).

(missing links right now).

Reply:
Agree, did a regex search replace now, but would be easier if hass-release did it